src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=online-book-store
+spring.datasource.url=jdbc:mysql://localhost:3306/onlinebookstore
+spring.datasource.username=root
+spring.datasource.password=Qwerty
```

Comment:
let's not push credentials to the remote git, and just leave those field empty or with some placeholder

Reply:
Replace with environment variables.
Done